CMakeLists.txt

```diff
@@ -429,6 +428,16 @@ if(USE_MAYA)
   include(OpenVDBMayaSetup)
 endif()
 
+# Configure malloc library. Use Tbbmalloc for Maya and Jemalloc for non-Maya.
+
+if(CONCURRENT_MALLOC STREQUAL "Auto")
+  if(USE_MAYA)
+    set(CONCURRENT_MALLOC "Tbbmalloc")
+  else()
+    set(CONCURRENT_MALLOC "Jemalloc")
+  endif()
+endif()
+
```

Comment:
Can you add a warning here if CONCURRENT_MALLOC is something other than "Auto", "Tbbmalloc", "Jemalloc", or "None", and then maybe set it to either "Auto" or "None"?

Reply:
I made it fallback on Auto.